### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

**Comment by jeffmendoza** [comment]
cc @mmaraya

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
enforce security policies. Its goal is to be able to detect any GitHub setting
or repository file contents that may be risky or not follow security best
practices. If Allstar finds a repository to be out of compliance, it will
```

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
contributing new policies is intended to be easy.
```

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
repositories. Follow the [quick start
```

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
For more details on advanced configuration, see below.
```

Not sure about this, but the original wording seems verbose.

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
Allstar can be enabled on individual repositories at the app level, with
the option of enabling or disabling each security policy individually. For
organization-level configuration, create a repository named `.allstar` in your
organization. Then create a file called `allstar.yaml` in that
repository.
```

File: README.md
Comment (loosebazooka) [inline README.md:102]: Not sure what "Allstar" is referring to? maybe `.allstar`?

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
policies on individual repositories. A policy will not take action unless
it is enabled **and** Allstar is enabled as a whole.
```
both it is... read weird, I'm not sure this suggestion is any better though

File: README.md
Comment (loosebazooka) [inline README.md]: link to scorecard checks available?

File: README.md
Comment (loosebazooka) [inline README.md]:
```suggestion
README.md with some information on how Allstar is being used in your
```

**Review by loosebazooka — COMMENTED** [review]
Some formatting stuff.

File: README.md
Comment (inferno-chromium) [inline README.md]: to continously monitor and detect ?

File: README.md
Comment (inferno-chromium) [inline README.md]: or do not

File: README.md
Comment (inferno-chromium) [inline README.md]: issue or restore secure settings (in explicitly enabled enforcement mode)

File: README.md
Comment (inferno-chromium) [inline README.md]: [OpenSSF](https://openssf.org/) organization, as part of

File: README.md
Comment (inferno-chromium) [reply README.md]: +1

File: README.md
Comment (inferno-chromium) [reply README.md]: Maybe start with [Install AllStar GitHub App]

File: README.md
Comment (inferno-chromium) [reply README.md]: +1

File: README.md
Comment (inferno-chromium) [inline README.md]: s/may be explicitly listed/would need to be explicitly added

File: README.md
Comment (inferno-chromium) [inline README.md]: add opt-out strategy (recommended) ?

File: README.md
Comment (inferno-chromium) [inline README.md]: s/In/At

File: README.md
Comment (inferno-chromium) [reply README.md]: both is weird
i like Appu's suggestion or split into two bullet points.

File: README.md
Comment (inferno-chromium) [inline README.md]: Add some word and then hyperlink these two.

File: README.md
Comment (inferno-chromium) [inline README.md]: s/should be closed/will be automatically closed

File: README.md
Comment (inferno-chromium) [inline README.md]: s/, not yet/, but not yet
remove , between "implemented, actions"

Comment (inferno-chromium) [reply README.md]: definitions will be added in the future

File: README.md
Comment (inferno-chromium) [inline README.md]: s/could/would send an email to the repo administrator(s)

File: README.md
Comment (inferno-chromium) [inline README.md]: s/could/world

File: README.md
Comment (inferno-chromium) [inline README.md]: merged if the check fails

File: README.md
Comment (inferno-chromium) [inline README.md]: "this fix action is not yet implemented" can be just added in end, rather than repeating in every policy.

File: README.md
Comment (inferno-chromium) [inline README.md]: s/Typically only organization members have this access./Only organization member should have this access, as otherwise untrusted members can change admin level settings and commit malicious code.

File: README.md
Comment (inferno-chromium) [inline README.md]: typos and confusing recommendation, are we putting developement instruction in docs, can exclude those here and instead put these in github issues for these features.

File: README.md
Comment (inferno-chromium) [reply README.md]: Instead add a line on how to manually fix these checks, that part is important as these users wont know. (in each of the checks).

**Review by inferno-chromium — DISMISSED** [review]
LGTM, thanks for writing this. Left some nit comments, can commit after that.

small things
1) should ask kara to take a doc improvement pass early next week (this cl does not need to wait, it is after this cl lands).
2) the doc does not say anything about app permissions. need to relieve people that it only read settings and create github issue (no other write settings needed unless enabling enforcement mode which will come in future).

File: README.md
Comment (laurentsimon) [inline README.md]: not sure what 'app level' means. individual repository level?

File: README.md
Comment (laurentsimon) [inline README.md:39]: maybe we should have 2 distinct section here, one for individual repo and one for org-level. Can provide a simple example of enabling/disabling for individual repo, like you do for org-level.

File: README.md
Comment (laurentsimon) [inline README.md]: in the quick installation guide, you could point to a repo example (container tool one)

File: README.md
Comment (laurentsimon) [inline README.md:122]: where are the logs visible?

**Review by laurentsimon — DISMISSED** [review]

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md:39]: The "Repository Override" section below covers that.

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: sounds good, done.

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md:102]: Referring to the app here. Just trying to say that some repos will need to opt-out of Allstar or individual policies, and using GitHub PRs on `.allstar` is a good way to have control over approval.

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md:122]: next line: "Logs are currently
  only visible to the app operator, plans to expose these are under discussion."

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: removed.

File: README.md
Comment (jeffmendoza) [reply README.md]: done

File: README.md
Comment (jeffmendoza) [reply README.md]: The Allstar app can be enabled/disabled overall, and then individual policies can be enabled/disabled.

File: README.md
Comment (jeffmendoza) [reply README.md]: done

**Comment by jeffmendoza** [comment]
Comments addressed, please re-approve =)

**Review by laurentsimon — APPROVED** [review]

**Pull request merged**
